wolfssl/wolfcrypt/sha.h

```diff
@@ -151,8 +151,8 @@ struct wc_Sha {
     #else
     word32  digest[WC_SHA_DIGEST_SIZE / sizeof(word32)];
     #endif
-#endif
     void*   heap;
+#endif
```

Comment:
Why exclude heap from ohter builds here? Have we confirmed that building with stm32 or silabs will compile okay with this change?

Reply:
good catch!  that was _not_ supposed to be in the changeset.  will fix.

Reply:
I've reapplied @danielinux 's commit that made those changes -- they look like the right idea.